apps/price_pusher/.gitignore

```diff
@@ -1,3 +1,4 @@
 docker-compose.yaml
 price-config.yaml
 lib
+mnemonic
```

Comment:
many of the sample testnet config files use `/mnemonic` as the private key file so adding it here to prevent people from accidentally comitting it

Reply:
thanks